model/Distribution.js

```diff
@@ -15,8 +15,26 @@ const distributionSchema = new Schema({
   current: { type: Number, default: 0 },
   satisfied: { type: Boolean, default: false },
   courses: [{ type: Schema.Types.ObjectId, ref: "Course" }],
+  description: { type: String, required: true },
+  criteria: { type: String, required: true },
+  min_credits_per_course: { type: Number, required: true },
+  fine_requirements: [
+    {
+      description: { type: String, required: true },
+      required_credits: { type: Number, required: true },
+      criteria: { type: String, required: true },
+      exception: { type: String },
```

Comment:
the frontend has not implemented the exception feature and is only relying on the "^NOT^" in the criteria string. This field should be removed.

Reply:
The latest commit [rename exclusive to double_count, in line with frontend obj](https://github.com/uCredit-Dev/uCredit-API/pull/44/commits/216957e3c5193873e26fdd98cf63087c44920752)
[216957e](https://github.com/uCredit-Dev/uCredit-API/pull/44/commits/216957e3c5193873e26fdd98cf63087c44920752) does this ^^